blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/rest/ActionProxiesImpl.java

```diff
@@ -86,13 +91,17 @@ public Link getLink() {
             @Override
             public boolean apply(Action action) {
                 return action != null
-                    && (action.getClass().getAnnotation(ExportedBean.class) != null || alwaysAllowAction.apply(action))
-                    && !BANNED_ACTIONS.contains(action.getClass().getName());
+                    && (action.getClass().getAnnotation(ExportedBean.class) != null || alwaysAllowAction.apply(action));
```

Comment:
Don't you still need `BANNED_ACTIONS` until the client explicitly requests actions?